From 3904963376ccc4015a542ada18335933afeea252 Mon Sep 17 00:00:00 2001
From: Robin Lee <cheeselee@fedoraproject.org>
Date: Wed, 30 Jun 2021 09:43:15 +0800
Subject: [PATCH] Rename the desktop entry file to match the application ID

GNOME Shell requires this to load application icon and prepare
sandboxing[1].

[1] https://wiki.gnome.org/Projects/GnomeShell/ApplicationBased
---
 .../{kchmviewer.desktop => net.kchmviewer.kchmviewer.desktop}     | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename packages/{kchmviewer.desktop => net.kchmviewer.kchmviewer.desktop} (100%)

diff --git a/packages/kchmviewer.desktop b/packages/net.kchmviewer.kchmviewer.desktop
similarity index 100%
rename from packages/kchmviewer.desktop
rename to packages/net.kchmviewer.kchmviewer.desktop
-- 
2.31.1

